makefile

```diff
@@ -80,12 +105,12 @@ export
 
 all: $(TARGET)
 
-include makefile.depend
+include $(REGCOIL_PATH)/makefile.depend
```

Comment:
Would it work for you if we kept this line the way it was? Introducing REGCOIL_PATH here means that all systems, not just PPPL, now need to define this variable. (Otherwise make looks for makefile.depend in /). makefile.depend should be in the same directory as this makefile, so I would think you would not need an absolute path here.